google-github/templates/workload-cluster/provider-config/providerconfig.yaml

```
@@ -14,28 +14,27 @@ spec:
         required_providers {
           google = {
             source = "hashicorp/google"
-            version = "5.35.0"
+            version = "~> 6.11.0"
```

```suggestion
            version = "~> 6.11.0"
          }
          google-beta = {
            source  = "hashicorp/google-beta"
            version = "~> 6.11.0"
```

Do we also need to specify the `google-beta` provider in here too?

---

yes crossplane complained of if not being present